docs/Features.md

```diff
@@ -0,0 +1,216 @@
+```
+
+Now, to say the default "Hello" to many people:
+
+```rust
+==> Say @{} to @{Jim}.
+==> Say @{} to @{Tom}.
+==> Say @{} to @{Angela}.
+```
+~~~
+
+These features allows for more flexibility when writing macros, as well as possibly making the intent
+clearer.
+
+## Extracted comments
+
+By default, the comment extraction sequence is set to `//`, purely for familiarity. Any text after
+(and including) this sequence is extracted from the code block, and not rendered to the tangled
+source code. Note that, since the comments are removed completely when compiling, they do not have
+to use the actual line comment indicator from you programming language. In fact, it may be better to
+choose a sequence that is *not* the regular comment indicator so that you can still have comments in
+your tangled code output.
+
+Now that these comments are extracted, it is possible to handle them differently in the weaved
+documentation file. Though some formats do not support any special behaviour, and simply write these
+comments back into the code, some are able to provide special rendering. In particular, the standard
+Markdown and HTML styles are able to render extracted comments in `<aside>` tags, which can then be
+rendered nicely using CSS.
+
+See the HTML example for an example of one way to render the extracted comments.
+
+## Named entrypoints
+
+By default, the entrypoint of the program is always the unnamed code block. However, this limits the
+output of one input file to always be the same source code. It also means that you can't have a name
+on the entrypoint in the documentation, which could be useful.
+
+To get around this, an entrypoint name can be passed to Yarner on the command line. Then, instead
+of starting at the unnamed code block, it will start at the code block with this name.
+
+Note that if you use a named entrypoint, there is no way to reference the unnamed code blocks as
+macros. You can, however, use the unnamed code blocks to provide examples, for example, to the
+readers of the documentation, so they are still useful.
+
+## Multiple Files, Multiple Entrypoints
+
+By naming code blocks with prefix `file:` followed by a relative path, multiple code files can be created
+from one source file. Each code block with the `file:` prefix is treated as a separate entry point.
+
+~~~md
+```rust - file:src/lib.rs
+fn say_hello() {
+    println!("Hello Literate Programmer!");
+}
+```
+~~~
+
+## File Transclusions
+
+Yarner supports file transclusion to allow for more structured or modular project sources.
+*This feature is currently only supported for Markdown.*
+
+Here, the content of file `src/main.rs.md` would be pulled into the document
+before compilation:
+
+```md
+@{{[src/main.rs](src/main.rs.md)}}
+```
+which would render as a link in the sources file. Or simply:
+```md
+@{{src/main.rs.md}}
+```
+which would not render as a link.
+
+Unnamed entrypoints are renamed to file entrypoints during transclusion.
```

Comment:
I think these last two paragraphs would benefit from a worked example now that the space limitation is gone.

Reply:
For both, renaming entrypoints and "no own entrypoints"?

Reply:
I think of them would suffice to illustrate what is happening.